### ISSUE TITLE
This project cannot be packaged because packaging has been disabled

### ISSUE THREAD

**Issue by isaacrlevin** [xoofx/dotnet-releaser#12]
I am trying to run dotnet-releaser on my Open Source project [PresenceLight](https://github.com/isaacrlevin/presencelight) and when I run 

```
 cd .\Personal\presencelight\src\PresenceLight.Web\
dotnet-releaser build --force .\dotnet-releaser.toml
dotnet-releaser new --project .\PresenceLight.Web.csproj
```

I get this error

```
C:\Program Files\dotnet\sdk\6.0.200-preview.22055.15\Sdks\NuGet.Build.Tasks.Pack\build\NuGet.Build.Tasks.Pack.targets(203,6): warning : This project cannot be packaged because packaging has been disabled. Add <IsPackable>true</IsPackable> to the project file to enable producing a package from this project. [D:\dev\Personal\presencelight\src\PresenceLight.Web\PresenceLight.Web.csproj]
fail: dotnet-releaser[6]
      Failed to build nuget packages or no packages were found.
```

The app I am trying to run is a web project, and I assume that this app only works with Class libraries by default? I would rather not add `IsPackable` to my project file. Could you check the SDK for Web or something else to detect web projects that do not need to be packed?

**Comment by xoofx** [comment]
Could you try to disable nuget?

```toml
[nuget]
publish = false
```

**Comment by xoofx** [comment]
Though `dotnet-releaser` still requires that you provide some package information (e.g description or PackageId actually)

I will have to double check if I produce an error in case this is not present and you are requesting to create a package (e.g `deb`)

**Comment by isaacrlevin** [comment]
It looks to be working just with that toml change
![image](https://user-images.githubusercontent.com/8878502/152389031-1a4e8636-f664-48dc-817b-7842854c6fde.png)

**Comment by xoofx** [comment]
I will keep this issue around. It can detect easily if the package is packable and skip this step entirely.

**Comment by xoofx** [comment]
Fixed by commit d523f1c. Should be part of up-coming `0.1.9+`
